src/main/java/io/strimzi/kafka/bridge/http/HttpAdminBridgeEndpoint.java

```diff
@@ -52,7 +52,7 @@ public class HttpAdminBridgeEndpoint extends HttpBridgeEndpoint {
      * @param context the HTTP bridge context
      */
     public HttpAdminBridgeEndpoint(BridgeConfig bridgeConfig, HttpBridgeContext context) {
-        super(bridgeConfig, null);
+        super(bridgeConfig);
```

Comment:
this is already in main. This class is not involved at all in the text embedded format anyway.